src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,37 @@
+projectId=cdf-athena
+dataset=test_automation
+
+gcsCSVFileFormat=csv
+gcsTSVFileFormat=tsv
+gcsAvroFileFormat=avro
+gcsBlobFileFormat=blob
+gcsParquetFileFormat=parquet
+gcsDelimitedFileFormat=delimited
+gcsJsonFileFormat=json
+gcsTextFileFormat=text
+gcsInvalidBucketName=invalidBucketName
+gcsCsvBucket=gs://cdf-athena-test/EmployeeTest.csv
```

Comment:
These variables are pointing to files inside the `cdf-athena-test` bucket and not to bucket. Please rename them to file like `gcsCsvFile` etc.

Reply:
Renamed